pkg/device-plugin/nvidiadevice/nvinternal/info/version.go

```diff
@@ -57,3 +57,8 @@ func GetVersionString(more ...string) string {
 	v := append(GetVersionParts(), more...)
 	return strings.Join(v, "\n")
 }
+
+// GetVersion returns the version of the binary.
+func GetVersion() string {
+	return version
```

Comment:
This version has already been injected at compile time.

https://github.com/Project-HAMi/HAMi/blob/master/Makefile#L38